modules/cluster-logging-loki-deploy.adoc

```diff
@@ -0,0 +1,127 @@
+// Module is included in the following assemblies:
+//cluster-logging-loki.adoc
+:_content-type: PROCEDURE
+[id="logging-loki-deploy_{context}"]
+= Deploying the LokiStack
+You can use the {product-title} web console to install the LokiOperator.
+
+.Prerequisites
+
+* {logging-title-uc} Operator 5.5
+* S3 bucket for log storage
+
+To install the LokiStack using the {product-title} web console:
+
+. Install the LokiOperator:
+
+.. In the {product-title} web console, click *Operators* -> *OperatorHub*.
+
+.. Choose  *LokiOperator* from the list of available Operators, and click *Install*.
+
+.. Under *Installation Mode*, select *All namespaces on the cluster*.
+
+.. Under *Installed Namespace*, select *openshift-operators-redhat*.
++
+You must specify the `openshift-operators-redhat` namespace. The `openshift-operators`
+namespace might contain Community Operators, which are untrusted and could publish
+a metric with the same name as an {product-title} metric, which would cause
+conflicts.
+
+.. Select *Enable operator recommended cluster monitoring on this namespace*.
++
+This option sets the `openshift.io/cluster-monitoring: "true"` label in the Namespace object.
+You must select this option to ensure that cluster monitoring
+scrapes the `openshift-operators-redhat` namespace.
+
+.. Select an *Approval Strategy*.
++
+* The *Automatic* strategy allows Operator Lifecycle Manager (OLM) to automatically update the Operator when a new version is available.
++
+* The *Manual* strategy requires a user with appropriate credentials to approve the Operator update.
+
+.. Click *Install*.
+
+.. Verify that you installed the LokiOperator. Visit the *Operators* → *Installed Operators* page and look for "LokiOperator."
+
+.. Ensure that *LokiOperator* is listed with *Status* as *Succeeded* in all the projects.
++
+. Create a Secret YAML file that uses the `access_key_id` and `access_key_secret` fields to specify your base64-encoded AWS credentials. For example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: logging-loki-s3
+  namespace: openshift-logging
+data:
+  access_key_id: QUtJQUlPU0ZPRE5ON0VYQU1QTEUK
+  access_key_secret: d0phbHJYVXRuRkVNSS9LN01ERU5HL2JQeFJmaUNZRVhBTVBMRUtFWQo=
+  bucketnames: czMtYnVja2V0LW5hbWU= <1>
+  endpoint: aHR0cHM6Ly9zMy5ldS1jZW50cmFsLTEuYW1hem9uYXdzLmNvbQ== <2>
+----
+<1> Base64-encoded bucket name
+<2> Base64-encoded S3 API endpoint
++
+. Create `LokiStack` Custom Resource
++
+[source,yaml]
+----
+  apiVersion: loki.grafana.com/v1
+  kind: LokiStack
+  metadata:
+    name: logging-loki
+    namespace: openshift-logging
+  spec:
+    size: 1x.small
+    storage:
+      schemas:
+      - version: v12
+        effectiveDate: 2022-06-01
+      secret:
+        name: logging-loki-s3
+        type: s3
+    storageClassName: gp2
+    tenants:
+      mode: openshift-logging
```

Comment:
Can we add a note regarding storage schema version. If the user has not added any storage schema in the CR, then storage schema is created with v11.
To upgrade to v12, we need to add another new section in the `schemas` list.

For e.g. 
```
    storage:
      schemas:
      - version: v11
        effectiveDate: 2020-06-01
      - version: v12
      	effectiveDate: <future date>
```

Reply:
I am open to adding it later, but we don't mention storage schema at all anywhere else in the logging docs, is there somewhere else in the OpenShift Docs that we might be able to link to or pull from to contextualize that option for users?

Reply:
Sounds good! this schema version is referenced to Loki only.
Reference: https://grafana.com/docs/loki/latest/operations/storage/schema/

Reply:
Maybe the Grafana docs?
https://grafana.com/docs/loki/latest/operations/storage/schema/#changing-the-schema

Reply:
Added to additional resources.